src/VisualStudio/TestSetup/VSPackage.resx

```diff
@@ -0,0 +1,122 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Doesn't look like there's any resources in this...? Or is this being used as a target for VSCT merging?